### PR TITLE
new garlic mutation: hairy garlic

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

hairy garlic! it gives you plant themed hair when you eat it.

right now i have very long strawberry fringe, and unkept watermelon, i'm gonna add a couple more :)

also remind me to sprite the hairy garlic

## Why It's Good For The Game

garlic has ZERO MUTATIONS. wild. it also gives me a chance to make a couple stupid hair styles that you can't get roundstart, pretty interesting way for botany to go

## Changelog
:cl:
add: garlic has a new mutation!
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
wah, when i eat this GARLIC, it makes my head into WATERMELON

**Comment by tralezab** [comment]
have sex

**Comment by Mickyan** [comment]
the greatest weapon against bald vampires

**Comment by bobbahbrown** [comment]
great pr keep up the fantastic work!

Sent from my Samsung Smart Fridge

**Comment by HomocidlePancakes** [comment]
I about to ruin this for you in one phrase...

...Pale scrotum

File: code/modules/hydroponics/grown/garlic.dm [hunk 1]
```diff
@@ -20,3 +21,47 @@
...
+	potency = 25
+	growthstages = 3
+	growing_icon = 'icons/obj/hydroponics/growing_vegetables.dmi'
+	mutatelist = list(/obj/item/seeds/garlic/hairy)
+	reagents_add = list(/datum/reagent/consumable/garlic = 0.15, /datum/reagent/consumable/nutriment = 0.1)
+
+/obj/item/reagent_containers/food/snacks/grown/garlic/hairy
+	seed = /obj/item/seeds/garlic/hairy
+	name = "hairy garlic"
+	desc = "Delicious, but with a potentially overwhelming odor AND disgusting hair on it like you dropped it on the floor and didn't meet the 5 second rule in time."
```
Comment (ATH1909) [inline code/modules/hydroponics/grown/garlic.dm]: add a comma after "it"? or is how it is currently grammatically correct?

File: code/modules/hydroponics/grown/garlic.dm [hunk 2]
```diff
@@ -20,3 +21,47 @@
...
+	icon_state = "hairygarlic"
+	tastes = list("garlic" = 1, "hair" = 3)
+	wine_power = 5
+
+/obj/item/reagent_containers/food/snacks/grown/garlic/hairy/Initialize(mapload, obj/item/seeds/new_seed)
+	. = ..()
+	bitesize = reagents.total_volume //one bite!
```
Comment (ATH1909) [inline code/modules/hydroponics/grown/garlic.dm]: make sure that the order of operations is correct with this and the densified chems trait

File: code/modules/hydroponics/grown/garlic.dm [hunk 3]
```diff
@@ -20,3 +21,47 @@
...
+
+/obj/item/reagent_containers/food/snacks/grown/garlic/hairy/On_Consume(mob/living/eater)
+	if(ishuman(eater))
+		var/mob/living/carbon/human/H = eater
+		to_chat(H, "<span class='notice'>Your hair shapes into something... fruity?!</span>")
```
Comment (ATH1909) [inline code/modules/hydroponics/grown/garlic.dm]: *reshapes

also, what happens if they were bald before?

**Comment by FantasticFwoosh** [comment]
The year is 2020 and we still dont have bee-beards.

This is ok too, not much love for facial hair bee'ing seen though.

**Comment by tralezab** [comment]
@FantasticFwoosh if you make it, i will gladly include it in the pr

File: code/modules/hydroponics/grown/garlic.dm (hunk 2 above)
Comment (tralezab) [reply code/modules/hydroponics/grown/garlic.dm]: made a workaround

File: code/modules/hydroponics/grown/garlic.dm (hunk 1 above)
Comment (tralezab) [reply code/modules/hydroponics/grown/garlic.dm]: maybe runon

**Comment by tralezab** [comment]
i have no sprite game

**Pull request closed without merging**